pipeline.yml

```diff
@@ -7,7 +7,7 @@ jobs:
   - task: run-compliance-masonry
     config:
       platform: linux
-      image: docker:///geramirez/compliance-masonry-cli-build#0.3
+      image: docker:///geramirez/compliance-masonry-cli-build#0.4
```

Comment:
Where is this Dockerfile? Can we move the Docker repository to https://hub.docker.com/u/18fgsa/?